backend/typescript/rest/camperRoutes.ts

```diff
@@ -87,6 +87,19 @@ camperRouter.get("/refund-confirm/:chargeId", async (req, res) => {
   }
 });
 
```

Comment:
Added the comment, and yeah after some thought I think it is better to just remove the `/campers-removal/` part in case we might use this endpoint for purposes other than removing campers lol. Done.